packages/eslint/package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "@swisspost/design-system-eslint",
+  "version": "1.0.0",
+  "description": "A set of custom ESLint rules to help keep projects using the Swiss Post Design System up-to-date.",
+  "author": "Swiss Post <design-system@post.ch>",
+  "license": "Apache-2.0",
+  "private": true,
+  "main": "dist/index.ts",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/swisspost/design-system",
+    "directory": "packages/eslint-plugin"
+  },
+  "homepage": "https://design-system.post.ch",
+  "bugs": {
+    "url": "https://github.com/swisspost/design-system/issues"
+  },
+  "publishConfig": {
+    "directory": "./dist",
+    "access": "public",
+    "linkDirectory": true
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "pnpm clean && tsc -p tsconfig.build.json",
+    "start": "pnpm build --watch",
+    "inspect": "wait-on dist && eslint --c eslint.play.js --inspect-config",
+    "play": "npm-run-all --parallel --print-label start inspect",
+    "lint": "eslint",
+    "lint:fix": "eslint --fix",
+    "test": "jest",
+    "test:watch": "jest --watchAll"
```

```suggestion
    "unit": "jest",
    "unit:watch": "jest --watchAll"
```
Otherwise, they will not being executed within our unit-tests.yaml workflow on github.